### PR TITLE
add a cronjob crud example

### PR DESCRIPTION
**Pull request by xg0719**
#### What this PR does / why we need it:
show people basic cronjob cases
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
NONE

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by xg0719** [comment]
> [CLA Missing ID](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/20677673/72473727/1815/#/?version=2) [CLA Not Signed](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/20677673/72473727/1815/#/?version=2)
> 
> * x The commit ([5645c57](https://github.com/kubernetes-client/python/commit/5645c578599b7dc953fd9926e0e92bdcf0e383fe)). This user is missing the User's ID, preventing the EasyCLA check. [Consult GitHub Help](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) to resolve.For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

**Comment by roycaihw** [comment]
please sign the CLA
/assign

File: examples/cronjob_crud.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        print(f"{name} doesn't exists, please enter a new one!")
+
+
+def get_cronjob_list(namespace='default'):
+    v1 = client.BatchV1beta1Api()
+    ret = v1.list_namespaced_cron_job(namespace=namespace, pretty=True, _preload_content=False)
+    cron_job_list = json.loads(ret.data)
+    print(f'cronjob number={len(cron_job_list["items"])}')
+    # for cron in cron_job_list['items']:
+    #     print(json.dumps(cron))
```
Comment (roycaihw) [inline examples/cronjob_crud.py:63]: remove?

**Review by roycaihw — COMMENTED** [review]
please sign the CLA

**Pull request closed without merging**